persist/dispatcher.go

```diff
@@ -14,16 +14,18 @@
 
 package persist
 
+//go:generate mockgen -destination=./mocks/mock_dispatcher.go -package=mocks github.com/casbin/casbin/v2/persist Dispatcher
+
 // Dispatcher is the interface for Casbin dispatcher
 type Dispatcher interface {
-	// AddPolicies adds policies rule to all instance.
-	AddPolicies(sec string, ptype string, rules [][]string) error
-	// RemovePolicies removes policies rule from all instance.
-	RemovePolicies(sec string, ptype string, rules [][]string) error
+	// AddPolicy adds policies rule to all instance.
```

Comment:
it's policies, not policy.

Reply:
@hsluoyz Because there is no `AddPolicy`, so I use this.